app/configurator/components/chart-options-selector.tsx

```diff
@@ -688,6 +691,10 @@ const ChartFieldColorComponent = ({
     "color",
     "componentIri",
   ]) as string | undefined;
+  const colorComponent = [
+    ...dataSetMetadata.dimensions,
+    ...dataSetMetadata.measures,
+  ].find((d) => d.iri === colorComponentIri);
```

Comment:
At some point, we'll want to make a useDimensionByIri hook backed by a provider, we do too much of this pattern everywhere 😅